tfhe/src/shortint/server_key/mod.rs

```diff
@@ -534,6 +534,22 @@ impl ServerKey {
             engine.create_trivial_assign(self, ct, value).unwrap()
         })
     }
+
+    pub fn  bootstrapping_key_size_elements(&self) -> usize {
+        self.bootstrapping_key.as_view().data().as_ref().len()
+    }
+
+    pub fn bootstrapping_key_size_bytes(&self) -> usize {
+        self.bootstrapping_key.as_view().data().as_ref().len() * std::mem::size_of::<concrete_fft::c64>()
```

Comment:
let's call bootstrapping_key_size_elements here to avoid having to update potential formulas twice :)